bin/reth/src/node/mod.rs

```diff
@@ -115,6 +115,11 @@ impl Command {
         info!(target: "reth::cli", "Connecting to P2P network");
         let netconf = self.load_network_config(&config, &db);
         let network = netconf.start_network().await?;
+
+        if !self.network.no_persist_peers {
+            load_peers(&self.network.peers_file, &network).await?;
+        }
```

Comment:
I would probably pull this in the `start_network` fn or make it a method on `NetworkHandle`

Reply:
Added this to `NetworkConfig`, and `load_network_config`.